src/components/composer/top_bar_composer/top_bar_composer.xml

```diff
@@ -0,0 +1,11 @@
+<templates>
+  <t t-name="o-spreadsheet-TopBarComposer" owl="1">
+    <div class="o-topbar-composer" t-att-style="containerStyle">
```

Comment:
Currently we have `div.o-grid/composer > div.o-composer-container > div.o-composer`

We can probably remove this `div.o-grid/composer` and give the container style as a new props to the composer. 2 issues : 

- I'm not 100% sure that we can do that for the grid composer, there's a t-ref that I _think_ we can do without, but not 100% sure.
    - I could do it only for the topbar, but I like the idea of having a symetrical structure for grid/topbar composers

- Idk if it's really better. It makes the DOM simpler (one less layer), but the code a bit harder to read, if we don't have one div per Component.

Reply:
lul :  doesn't really matter